LondonBritishLibrary/orient/BLorient576.xml

```diff
@@ -38,14 +38,121 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msContents>
                         <summary/>
                         
-                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="11r"/>
+                            <title type="complete" ref="LIT1850Malkea"/>  
+                            <note>The codicological structure of the manuscript cannot be discerned by the catalogue description. Several
```

Comment:
I find this sentence a little clumsy, frankly speaking, perhaps smth like: "The catalgue description does not indicate the structure of the manuscript that may be composed of several codicological units" (are there indications for that or it is an assumption?). Also, I would place this information elsewhere, I would not expect in a note ms_i1.

Reply:
I agree with your rewording and that the place is not ideal, do you have a better idea where to place this information? My priority would be maybe not the absolute ideal place from the point of view of the encoding, but one where under the current visualisation users have a good chance at spotting it. How about a `<note>` in support or history?